modules/utxo-lib/test/bitgo/wallet/WalletUnspent.ts

```diff
@@ -104,19 +101,8 @@ describe('WalletUnspent', function () {
 
     const psbt2 = createPsbtFromTransaction(
       tx,
-      unspents.map((u) => toPrevOutput<bigint>(u, network))
+      unspents.map((u) => ({ ...toPrevOutput<bigint>(u, network), prevTx: u.prevTx }))
     );
-    const nonWitnessUnspents = unspents.filter((u): u is WalletUnspent<bigint> & UnspentWithPrevTx =>
-      isUnspentWithPrevTx(u)
-    );
-    const txBufs = Object.fromEntries(
-      psbt2.getNonWitnessPreviousTxids().map((txid) => {
-        const u = nonWitnessUnspents.find((u) => parseOutputId(u.id).txid === txid);
-        if (u === undefined) throw new Error('No prevtx found');
-        return [txid, u.prevTx];
-      })
-    );
-    psbt2.addNonWitnessUtxos(txBufs);
```

Comment:
@brandonblack do you see any issue with this?

Reply:
If we're going to do this, we should make `createPsbtFromTransaction` populate the `nonWitnessUtxo`s from the prevTx. We don't currently, and the `TxOutput` type doesn't have a field for it.

Reply:
I actually did add support for it here: https://github.com/BitGo/BitGoJS/commit/b1c91bcda40e4a8caed8302ff0206d3dfce85f9f